### PR TITLE
feat: added second argument to deleteIn/deleteOut to specify exact triple

### LINKED ISSUE #24: Cannot delete specific objects
It is only possible to `cf.deleteOut(predicates)` which will remove all triples.

The methods should be extended to take a second param with the object(s) to delete

```
cf.deleteOut(predicate, objects)
```

### PR DESCRIPTION
**Pull request by bergos**
fixes #24

### CODE REVIEW

**Review by l00mi — APPROVED** [review]

File: lib/Clownface.js [hunk 1]
```diff
@@ -389,12 +389,14 @@ class Clownface {
    * Deletes all quads where the current graph pointer contexts are the objects
    *
    * @param {Term|Term[]|Clownface|Clownface[]} predicates
+   * @param {Term|Term[]|Clownface|Clownface[]} subjects
```
Comment (tpluscode) [inline lib/Clownface.js:392]: This is optional, right?

```suggestion
   * @param {Term|Term[]|Clownface|Clownface[]} [subjects]
```

File: lib/Clownface.js [hunk 2]
```diff
@@ -403,12 +405,14 @@ class Clownface {
    * Deletes all quads where the current graph pointer contexts are the subjects
    *
    * @param {Term|Term[]|Clownface|Clownface[]} predicates
+   * @param {Term|Term[]|Clownface|Clownface[]} objects
```
Comment (tpluscode) [inline lib/Clownface.js:408]: Ditto

```suggestion
   * @param {Term|Term[]|Clownface|Clownface[]} [objects]
```

**Review by tpluscode — CHANGES_REQUESTED** [review]
Just JSDoc changes 👌

File: lib/Clownface.js (hunk 1 above)
Comment (bergos) [reply lib/Clownface.js]: You are right. The first argument is also optional, which was missing. Also fixed that.

**Review by tpluscode — APPROVED** [review]

**Pull request merged**